src/webui/www/private/scripts/client.js

```diff
@@ -161,6 +174,14 @@ window.addEvent('load', function() {
             updateMainData();
     };
 
+    setTagFilter = function(hash) {
+        selectedTag = hash;
+        localStorage.setItem('selected_tag', selectedTag);
+        highlightSelectedTag();
+        if (typeof torrentsTable.tableBody !== 'undefined')
```

Comment:
What's the difference between this `typeof` check  versus checking against `undefined` directly?